apps/website/src/app/atoms/icons/EmailIcon.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "../../../../node_modules/react";
+import * as React from "react";
```

Comment:
We're in Next.js context so we could probably remove this line altogether (across all files)